From a30e48be25ae196aea1fde233e9dfba1218f35b3 Mon Sep 17 00:00:00 2001
From: Bruno Haible <bruno@clisp.org>
Date: Thu, 29 Jan 2026 19:04:32 +0100
Subject: [PATCH] relocatable-prog-wrapper: Fix syntax error on Linux (regr.
 2025-12-09).

Reported by Erik Schnetter <schnetter@gmail.com>
at <https://savannah.gnu.org/bugs/?67987>.

* lib/progreloc.c (find_executable): Remove extraneous brace.
---
 gettext-runtime/gnulib-lib/progreloc.c | 1 -
 gettext-tools/gnulib-lib/progreloc.c   | 1 -
 2 files changed, 2 deletions(-)

diff --git a/gettext-runtime/gnulib-lib/progreloc.c b/gettext-runtime/gnulib-lib/progreloc.c
index bac7cd0..3596043 100644
--- a/gettext-runtime/gnulib-lib/progreloc.c
+++ b/gettext-runtime/gnulib-lib/progreloc.c
@@ -259,7 +259,6 @@ find_executable (const char *argv0)
     char *link = xreadlink (buf);
     if (link != NULL && link[0] != '[')
       return link;
-    }
     if (executable_fd < 0)
       executable_fd = open (buf, O_EXEC | O_CLOEXEC, 0);
   }
diff --git a/gettext-tools/gnulib-lib/progreloc.c b/gettext-tools/gnulib-lib/progreloc.c
index bac7cd0..3596043 100644
--- a/gettext-tools/gnulib-lib/progreloc.c
+++ b/gettext-tools/gnulib-lib/progreloc.c
@@ -259,7 +259,6 @@ find_executable (const char *argv0)
     char *link = xreadlink (buf);
     if (link != NULL && link[0] != '[')
       return link;
-    }
     if (executable_fd < 0)
       executable_fd = open (buf, O_EXEC | O_CLOEXEC, 0);
   }
-- 
2.53.0

